data/docs/traces-management/guides/correlate-traces-and-logs.mdx

```diff
@@ -0,0 +1,59 @@
+---
+date: 2025-07-01
+title: Correlate Traces and Logs
+id: correlate-traces-and-logs
+description: Learn how to establish correlation between logs and traces in SigNoz to enable seamless navigation from traces to logs and vice versa.
+---
+
+When you navigate from traces to logs and see "No logs found for this trace," it means your logs and traces are not properly correlated. This guide will help you establish the connection.
+
+## Automatic Correlation with OpenTelemetry SDK
+
+If you're using OpenTelemetry SDKs for instrumentation, they automatically inject trace context (trace_id and span_id) into your logs. Simply enable the logging instrumentation for your language and your existing logs will include trace correlation.
+
+Auto-instrumentation is available for these languages - Java, .NET, Python, Go, Node.js, and PHP. Check out our [instrumentation documentation](https://signoz.io/docs/instrumentation/) for auto-instrumenting your application.
```

Comment:
Most languages don't support auto injecting trace context to logs.
Also add more detail about this